### ISSUE TITLE
How do you feed the target class into GraphLIME?

### ISSUE THREAD

**Issue by zifanw** [WilliamCCHuang/GraphLIME#8]
Hey there,

Thanks for the implementation! Just a quick question, in the paper it says the GNN model is assumed to output the logit for the target class y_i, however, in the tutorial.ipynb and the code, I did not see a place that I can feed the target label into GraphLIME. Does the GraphLIME object assume that the model outputs only the logits of class i?

**Comment by WilliamCCHuang** [comment]
Hi @zifanw,

Thanks for pointing out that problem. Actually, I think that I might miss this detail when I implemented, and regarded y_i as the probabilities of all classes. I will ask the authors and fix it as soon as possible!
